test/cpp/jit/test_gpu.cpp

```diff
@@ -769,16 +769,107 @@ void testGPU_FusionTopoSort() {
 void testGPU_FusionTensor() {
   auto options = at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
 
-  auto tensor = at::randn({2, 3, 4, 5}, options);
-  auto sizes = tensor.sizes().vec();
-  auto tensor_type = TensorType::create(tensor);
-
   Fusion fusion;
   FusionGuard fg(&fusion);
 
-  auto fuser_tensor = new TensorView(tensor_type);
-  TORCH_CHECK(fuser_tensor->getDataType().value() == DataType::Float);
-  TORCH_CHECK(fuser_tensor->domain() != nullptr);
+  {
+    auto tensor = at::randn({2, 3, 4, 5}, options);
```

Comment:
I like the new test cases. If not too much trouble, it would be nice to separate them into separate test functions instead of multiple scopes inside the same test function.

Reply:
Are we just trying to speed up issue tracking? I can put some strings into `TORCH_CHECK` to identify failing tests.
The motivation to throw everything under the same function is simply because they are testing the same constructor of `TensorView` on a `TensorTypePtr`.

Not trying to push back on the comment and there certainly is not much trouble at all. But feels awkward to create multiple testing cases for a single constructor.